internal/devices/retrieve.go

```diff
@@ -47,27 +48,30 @@ func (c *Client) Retrieve() *cobra.Command {
 			}
 			cmd.SilenceUsage = true
 
+			include := []string{"Inner_example"}
+			exclude := []string{"Inner_example"}
+
 			if deviceID != "" {
-				device, _, err := c.Service.Get(deviceID, nil)
+				device, _, err := c.Service.FindDeviceById(context.Background(), deviceID).Include(include).Exclude(exclude).Execute()
```

Comment:
When the user passes the `--include` and/or `--exclude` command-line flags, the values they specify should go in the `include` and/or `exclude` arrays.  The `ListOptions` function takes care of that for `packngo`-based commands, but I don't think we have anything like that for `metal-go`.  We need to replicate that or we will break the `--include` and `--exclude`.

Reply:
I confirmed that `metal plans` lost support for these and other command-line flags; it looks like the other commands we've converted so far did not truly support those flags in the first place. https://github.com/equinix/metal-cli/issues/292